go/test/endtoend/vtgate/misc_test.go

```diff
@@ -133,6 +133,25 @@ func TestUnion(t *testing.T) {
 	assertMatches(t, conn, `(SELECT 1,'a' order by 1) union (SELECT 1,'a' ORDER BY 1)`, `[[INT64(1) VARCHAR("a")]]`)
 }
 
+// TestCheckConstraint test check constraints on CREATE TABLE
+// This feature is supported from MySQL 8.0.16 and MariaDB 10.2.1.
+func TestCheckConstraint(t *testing.T) {
+	conn, err := mysql.Connect(context.Background(), &vtParams)
```

Comment:
This is failing on CI because we are still running these tests against MySQL 5.7. We have to figure out a better way to run this.

In the meantime, would you mind just `t.Skip()` for now, and we can enable it when CI is ready to run it?